maven-modules/maven-build-lifecycle/src/test/java/com/baeldung/mavenlifecycle/CourseAppIT.java

```diff
@@ -0,0 +1,14 @@
+package com.baeldung.mavenlifecycle;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class CourseAppIT {
+
+    @Test
+    void givenIntegrationTest_whenGetCourse_ThenCourseShouldBePresent() {
+        CourseApp courseApp = new CourseApp();
```

Comment:
In both test classes, can we add a line between setup and assertion within our tests? For example,

` void whenGetCourse_ThenCourseShouldBePresent() {
        when(courseApp.getCourse()).thenReturn("Baeldung Spring Masterclass");

        assertEquals("Baeldung Spring Masterclass", courseApp.getCourse());
    }`

Reply:
this is addressed